src/utils/next.ts

```diff
@@ -75,6 +78,14 @@ const stripParams = (relativePath: string, params?: ParsedUrlQuery) => {
   return url.pathname + url.search;
 };
 
+const hasOrg = (reqWithSession: { session?: AppSession }, orgId: string) => {
+  return reqWithSession.session?.organizations?.find(
+    (org) => org === parseInt(orgId)
+  )
+    ? true
+    : false;
```

Comment:
```suggestion
  return Boolean(reqWithSession.session?.organizations?.find((org) => org === parseInt(orgId));
```

Rather than a turnary, you can use this "Boolean constructor" which will return true if a truthy input, and false if a falsy input

Reply:
This isn't formatted properly so you can't just approve it, sorry!

Reply:
Oooh fancy!! :D

Reply:
Nice! I saw this and had a similar idea, except I usually do `!!someTruthyOrFalsyExpression` instead of `Boolean(someTruthyOrFalsyExpression)`. I think wither way is more concise than ternary. 😊